db/view/view.cc

```diff
@@ -1070,6 +1070,32 @@ void view_update_builder::generate_update(clustering_row&& update, std::optional
     }
 }
 
+void view_update_builder::generate_update(static_row&& update, const tombstone& update_tomb,
+        std::optional<static_row>&& existing, const tombstone& existing_tomb) {
+    if (!update_tomb && update.empty()) {
+        throw std::logic_error("Empty materialized view updated");
```

Comment:
sounds weird - maybe you meant s/updated/update? In any case, please elaborate a little bit to indicate what's wrong, e.g. "a materialized view update cannot be empty"

Reply:
Changed the message to "A materialized view update cannot be empty"